Source/WebCore/css/CSSStyleSheet.cpp

```diff
@@ -73,6 +74,7 @@ class StyleSheetCSSRuleList final : public CSSRuleList {
 
     CSSStyleSheet* m_styleSheet;
 };
+DEFINE_ALLOCATOR_WITH_HEAP_IDENTIFIER(StyleSheetCSSRuleList);
```

Comment:
Why put the `define` up here and not by the `declare` macro?

Reply:
I don't know I either saw it somewhere or I thought it made sense given that any pair of DECLARE/DEFINE that happens in a header/cpp file combo means the define will always be after the class definition?

I don't think it matters.